libsolidity/interface/CompilerStack.cpp

```diff
@@ -1285,12 +1285,30 @@ void CompilerStack::generateEVMFromIR(ContractDefinition const& _contract)
 	//cout << yul::AsmPrinter{}(*stack.parserResult()->code) << endl;
 
 	// TODO: support passing metadata
-	auto result = stack.assemble(yul::AssemblyStack::Machine::EVM);
-	compiledContract.object = std::move(*result.bytecode);
-	// TODO: support runtimeObject
-	// TODO: add EIP-170 size check for runtimeObject
+	// TODO: use stack.assemble here!
+	yul::MachineAssemblyObject init;
+	yul::MachineAssemblyObject runtime;
+	std::tie(init, runtime) = stack.assembleAndGuessRuntime();
```

Comment:
```suggestion
	auto [init, runtime] = stack.assembleAndGuessRuntime();
```

Reply:
I did this, but then realised it is probably nicer to know the types here.

Reply:
ok